feat(content): Standardize tooltips (#9013)
endless-sky/endless-sky: data/_ui/tooltips.txt

tip "capture defense:"
	`Increases the "strength" of one crew member when defending against an attempt to capture this ship. Each crew member can only wield one weapon at a time, but they will use whichever weapons provide the highest power.`

tip "cargo:"
	`Tons of cargo this ship is carrying right now, out of its maximum.`

tip "cargo scan:"
	`Allows scanning your target ship's cargo from up to this distance away.`

tip "cargo scan power:"

> 	`Allows scanning your target ship's cargo. The range of this cargo scanner is proportional to the square root of this value.`

Reviewer: ```suggestion
	`Allows scanning of your target ship's cargo. The range of this cargo scanner is proportional to the square root of this value.`
```

	`Cost, in credits. If you have sold any used (depreciated) items today, you can buy them back for the same price you sold them for.`

tip "discharge protection:"

> 	`Protection provided shield discharge damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in discharge damage, while a total value of 11 only results in a 10 percent reduction.`

Reviewer 2: ```suggestion
	`Protection provided against shield discharge damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in discharge damage, while a total value of 11 only results in a 10 percent reduction.`
```

	`Heat generated per second when resisting discharge damage.`

tip "disruption protection:"
	`Protection provided against shield disruption damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in disruption damage, while a total value of 11 only results in a 10 percent reduction.`

tip "disruption resistance:"

> 	`Shield disruption damage resised per second.`

Reviewer 2: ```suggestion
	`Shield disruption damage resisted per second.`
```

	`Amount of credits you can be fined for carrying this outfit.`

tip "income:"
	`Amount of credits received per day when you own this outfit, even if it is not in active use.`
	
tip "inertia reduction:"
	`Reduces effective ship mass. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in inertia, while a total value of 11 only results in a 10 percent reduction.`

tip "ion protection:"
	`Protection provided against ion damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in ion damage, while a total value of 11 only results in a 10 percent reduction.`

tip "scramble protection:"

> 	`Protection provided against weapons that do scrambling damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in scramble damage, while a total value of 11 only results in a 10 percent reduction.`

Reviewer 3: This one didn't get the full protection tooltip treatment.

Author: ```suggestion
	`Protection provided against scrambling damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in scramble damage, while a total value of 11 only results in a 10 percent reduction.`
```

	`Heat generated per second when resisting scrambling damage.`

tip "jump speed:"
	`The maximum velocity the ship can jump at.`

tip "jump fuel:"
	`Fuel consumed when jumping between systems using this drive.`

tip "jump range:"
	`The maximum distance this drive can jump.`

tip "jump mass cost:"
	`Fuel consumed per 100 tons of mass when jumping. Additive with jump fuel.`

tip "jump base mass:"
	`Modifies the impact your ship's mass has on the jump mass cost. Ships below this mass will have a cheaper jump cost than the base jump fuel cost, while ships above this mass will have more expensive jumps.`

tip "leak protection:"

> 	`Protection provided against fuel leakage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in leak damage, while a total value of 11 only results in a 10 percent reduction.`

Reviewer 3: Should this be changed to "Protection provided against leak damage"?

Author: ```suggestion
	`Protection provided against leak damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in leak damage, while a total value of 11 only results in a 10 percent reduction.`
```

	`Heat generated per second when resisting leak damage.`

tip "maintenance costs:"
	`Credits spent per day to maintain this outfit. Even if a ship is parked or an outfit is in cargo, you still need to pay for its maintenance.`

tip "map:"
	`The number of star systems included in this map.`

tip "mass with no cargo:"
	`Mass of this ship when not carrying any cargo or fighters. Cargo increases mass, which reduces acceleration and turn rate and increases maximum heat before overheating.`

tip "mass:"
	`Mass, in tons. A ship's mass determines how fast it turns and accelerates and its maximum heat before overheating.`

tip "max speed:"

> 	`The maximum speed of the ship when using thrusters but no afterburners.`

Reviewer 3: Max speed actually can display the maximum speed while thrusters are active if there are no normal thrusters installed.

Also spotted that this says "the ship" while other tooltips around it say "this ship." Might be worth running through all the tooltips to standardize that. For these tooltips that only appear in the ship info sidebar, I think "this ship" makes sense.
```suggestion
	`The maximum speed of this ship when using thrusters. Excludes the effect of afterburners, unless there are no normal thrusters installed.`
```

Author: Both `the ship` and `this ship` are used quite often.

	`Increases the time required for other ships to perform an outfit scan of this ship. A value of one has the same influence on the time taken as adding one more ton of outfit space to scan.`

tip "outfit space added:"
	`Tons of general-purpose space this outfit provides.`

tip "outfit space needed:"
	`Tons of general-purpose space this outfit uses up. Some outfits also use up weapon space or engine space.`

tip "outfit space free:"
	`Tons of space this ship has for outfits (including weapons and engines).`

tip "overheat damage rate:"
	`Hull damage taken per second when overheated. Overheat damage scales with how far past the overheat threshold the ship is.`

tip "overheat damage threshold:"
	`Increases the percentage past overheated at which point overheat damage begins.`

tip "piercing protection:"

> 	`Protection provided against shield piercing damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in damage leak, while a total value of 11 only results in a 10 percent reduction.`

Reviewer 3: Piercing isn't an actual damage type, it's a modifier to shield and hull damage, so I don't think that calling it "shield piercing damage" makes sense. It's also not protection in the same way that the other protection attributes reduce the total amount of damage received, as piercing protection just bounces damage that would've hit the hull back to the shields. We could probably also use a different term than "damage leak," or maybe explain what is meant by that (although that may be redundant with reading piercing's tooltip).

Perhaps look to piercing resistance's tooltip.

Author: Piercing resistance/protection doesn't work with shield disruption, right? So if I say something like "protection provided against damage bypassing shields", that would be misleading.

~~This is hard.~~

```suggestion
	`Protection provided against shield piercing damage. Unlike other protection attributes, this attribute bounces the incoming hull damage to your shields, but doesn't reduce the overall damage dealt. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in damage leak, while a total value of 11 only results in a 10 percent reduction.`
```

I honestly can't think of a better name for piercing "damage".

Reviewer 3: Feels too wordy. I think the problem is the use of the word damage, because like I said, this isn't actually a damage type.
```suggestion
	`Protection provided against piercing. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in piercing, while a total value of 11 only results in a 10 percent reduction.`
```

	`Heat produced per second when firing reverse thrusters.`

tip "reverse thrusting shields:"
	`Shields displaced, consumed, or destroyed per second when firing reverse thrusters.`

tip "reverse thrusting hull:"

> 	`Hull that flakes off, breaks apart, or is otherwise compromised per second when firing reverse thrusters.`

Reviewer 3: I feel like these should just be simplified to "(Shield || Hull) damage taken per second..." instead of providing explanations for why the shields or hull might be getting used up. Such explanations are what the outfit's description should provide.

Author: `Shield damage dealt to the ship per second when firing reverse thrusters.`
Does this look good?

Also, is the accumulative damage for thrust worded properly?

Reviewer 3: That looks good. And I believe "accumulated" is used for various thrust costs because those are the DoT damage types. So when you fire a thruster that has 100 ion/sec, you're not losing 100 energy, you're accumulating 100 units of ionization that will decay over the next few seconds. I suppose that brings into question whether "accumulated" should be used for other firing DoT costs, or if the wording of other firing DoT costs should be used for the engine DoT costs.

	`Heat generated per second when resisting slowing damage.`

tip "solar collection:"
	`Energy produced depending on how far this ship is from the star at the center of the system.`

tip "solar heat:"
	`Heat produced depending on how far this ship is from the star at the center of the system.`

tip "tactical scan power:"

> 	`Allow viewing your current target's crew complement, fuel, heat, and energy levels when it is within range. The range is proportional to the square root of this value.`

Reviewer 2: ```suggestion
	`Allows viewing your current target's crew complement, fuel, heat, and energy levels when it is within range. The range is proportional to the square root of this value.`
```

tip "scrambling damage / second:"
	`Increases the target's chance of weapon failure. Only half as effective when the target's shields are up. Scrambling slowly wears off over time.`

tip "slowing damage / second:"
	`Reduces the target's turn rate, acceleration, and top speed. Only half as effective when the target's shields are up. Slowing wears off over time.`

tip "disruption damage / second:"
	`Allows weapon damage to "leak through" to the hull even if shields are up. Shield disruption wears off over time, and adding additional shield disruption becomes more difficult as damage leaks past shields instead of disrupting them.`

tip "discharge damage / second:"
	`Shield damage dealt over time by this weapon per second.`

tip "corrosion damage / second:"
	`Hull damage dealt over time by this weapon per second.`

tip "leak damage / second:"
	`Fuel damage dealt over time by this weapon per second.`

tip "burn damage / second:"
	`Heat damage dealt over time by this weapon per second.`

tip "% shield damage / second:"
	`Shield damage dealt per second, as a percentage of the target's total shield strength.`

tip "% hull damage / second:"

> 	`Hull damage dealt per second, as a percentage of the taget's total hull strength.`

Reviewer 2: ```suggestion
	`Hull damage dealt per second, as a percentage of the target's total hull strength.`
```

tip "% minable damage / second:"

> 	`Minable damage dealt per second, as a percentage of the target's total healt.`

Reviewer 2: ```suggestion
	`Minable damage dealt per second, as a percentage of the target's total health.`
```

Author: ~~Don't we have a spellcheck CI or something?~~

tip "% fuel damage / second:"

> 	`Fuel loss caused per second as a percentage of the the target's total fuel capacity. Only half as effective when the target's shields are up.`

Reviewer 2: ```suggestion
	`Fuel loss caused per second as a percentage of the target's total fuel capacity. Only half as effective when the target's shields are up.`
```

tip "turret turn rate:"
	`Degrees this turret can turn per second.`

tip "homing:"
	`This projectile's ability to track its target. Weapons with better homing can adjust to the target's speed to plot a more effective intercept course.`

tip "tracking:"
	`This projectile's ability to maintain its target lock.`

tip "optical tracking:"
	`This projectile's ability to maintain an optical target lock, which depends on the size of the target; negatively affected by any potential optical jamming equipement.`

tip "infrared tracking:"
	`This projectile's ability to maintain an infrared target lock, which depends on the target ship's heat level and distance from the missile; IR-guided missiles become more accurate the closer they are to their targets.`

tip "radar tracking:"
	`This projectile's ability to maintain a radar target lock; negatively affected by the target ship's radar jamming equipment.`

tip "piercing:"

> 	`Percentage of this weapon's damage that leaks through to the hull.`

Reviewer 3: ```suggestion
	`Percentage of this weapon's damage that leaks through to the hull while the target's shields are still active.`
```

tip "piercing resistance:"

> 	`Piercing damage resisted on all incoming projectiles, preventing damage from leaking through the shields to the hull.`

Reviewer 3: This should be up by piercing protection rather than under the weapon tooltips.

tip "shield damage / shot:"
	`Shield damage dealt per shot.`

tip "hull damage / shot:"
	`Hull damage dealt per shot.`

tip "minable damage / shot:"
	`Damage dealt to minable objects per shot, in lieu of hull damage.`

tip "fuel damage / shot:"
	`Fuel lost by the target per shot when hit by this weapon. Only half as effective when the target's shields are up. A negative amount means fuel is added.`

tip "heat damage / shot:"
	`Increases the target ship's heat by this amount per shot. Only half as effective when the target's shields are up.`

tip "energy damage / shot:"

> 	`Reduces the target ship's energy by this amount per shot, or half this amount if its shields are up.`

Reviewer 2: ```suggestion
	`Reduces the target ship's energy by this amount per shot. Only half as effective when the target's shields are up.`
```

tip "ion damage / shot:"
	`Increases the target's ionization by this amount per shot. Only half as effective when the target's shields are up. Ionization reduces the target's energy, and slowly wears off over time.`

tip "scrambling damage / shot:"
	`Increases the target's chance of weapon failure. Only half as effective when the target's shields are up. Scrambling slowly wears off over time.`

tip "slowing damage / shot:"
	`Reduces the target's turn rate, acceleration, and top speed. Only half as effective when the target's shields are up. Slowing wears off over time.`

tip "disruption damage / shot:"
	`Allows weapon damage to "leak through" to the hull even if shields are up. Shield disruption wears off over time, and adding additional shield disruption becomes more difficult as damage leaks past shields instead of disrupting them.`

tip "discharge damage / shot:"
	`Shield damage dealt over time by this weapon per shot.`

tip "corrosion damage / shot:"
	`Hull damage dealt over time by this weapon per shot.`

tip "leak damage / shot:"
	`Fuel damage dealt over time by this weapon per shot.`

tip "burn damage / shot:"

> 	`Fuel damage dealt over time by this weapon per shot.`

Reviewer 2: Is this supposed to be the same as the tooltip above?

Author: ```suggestion
	`Heat damage dealt over time by this weapon per shot.`
```

tip "firing discharge / shot:"
	`Shield damage dealt to your ship over time by this weapon per shot.`

tip "firing corrosion / shot:"
	`Hull damage dealt to your ship over time by this weapon per shot.`

tip "firing leak / shot:"
	`Fuel damage dealt to your ship over time per shot.`

tip "firing burn / shot:"
	`Heat damage dealt to your ship over time per shot.`

tip "% firing energy / shot:"
	`Energy consumed per shot, as a percentage of the ship's total energy capacity.`

tip "% firing heat / shot:"
	`Heat generated per shot, as a percentage of the ship's maximum heat capacity.`

tip "% firing fuel / shot:"
	`Fuel used or lost per shot, as a percentage of the ship's total fuel.`

tip "% firing hull / shot:"
	`Hull damage dealt to your ship per shot, as a percentage of your total hull strength.`

tip "% firing shields / shot:"
	`Shield damage dealt to your ship per shot, as a percentage of your total shield strength.`

tip "inaccuracy:"
	`Maximum inaccuracy of this weapon, in degrees. A value of one means the weapon's aim will be at most one degree off-target.`

tip "blast radius:"

> 	`Everything within this distance (including friendly ships) will be damaged when the weapon hits something.`

Reviewer 3: This reminds me: looks like we don't have a tooltip for "safe".

Author: We also don't have tooltips for "phasing", "gravitational" and probably a lot of other weapon attributes. I don't know which ones are displayed in the interface, though.

Reviewer 3: Just realized these are all weapon attributes, which don't appear unless we explicitly allow them to (which is the opposite of outfit attributes which appear by default unless we hide them), and these don't appear to be given tooltips.

tip "missile strength:"

> 	`This missile's resistance to anti-missile shots.`

Reviewer 3: Not all weapons with missile strength actually fire missiles, so maybe just refer to projectiles in the tooltip.
```suggestion
	`This projectile's resistance to anti-missile shots.`
```

tip "anti-missile:"

> 	`The power of the anti-missile shots. Higher values make it easier to destroy stronger missiles.`

Reviewer 3: ```suggestion
	`The power of the anti-missile shots. Higher values make it easier to destroy stronger projectiles.`
```

# Flight checks:
tip "overheating!"

> 	`This ship's outfits are producing so much heat that it will overheat the moment it takes off. You should either add more cooling or remove some of the outfits that are producing heat.`

Reviewer 3: Anything ending in an exclamation point is something you must resolve in order to actually leave the outfitter, so I think saying "must" instead of "should" for such tooltips makes sense.

Author: ```suggestion
	`This ship's outfits are producing so much heat that it will overheat the moment it takes off. You must either add more cooling or remove some of the outfits that are producing heat.`
```

	`This ship does not have enough fuel to provide energy to power its systems. You should increase the fuel capacity.`

tip "no thruster!"

> 	`This ship is unable to move. You should install some thrusters.`

Reviewer 3: ```suggestion
	`This ship is unable to move. You should install thrusters.`
```

tip "no steering!"

> 	`This ship is unable to turn. You should install some steering systems.`

Reviewer 3: ```suggestion
	`This ship is unable to turn. You should install steering systems.`
```
